clients/egui/src/account/mod.rs

```diff
@@ -192,8 +192,7 @@ impl AccountScreen {
                 }
 
                 if let Some(file) = wso.selected_file {
-                    self.tree.reveal_file(file, &self.core);
-                    ctx.request_repaint();
+                    self.tree.reveal_file(file, ctx);
```

Comment:
why is this repaint gone?

Reply:
it's moved inside the `reveal_file` function so that each time it's called a repaint happen